ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -0,0 +1,570 @@
+         $(document).ready(function(){
+
+            var sequence_numbers = {};
+            var all_sequences = {};
+            var interval_send = null;
+            var scheduled_send_media_clock_offset = null;
+            var interval_running_clock = null;
+
+            if (typeof(Storage) === "undefined") {
+               window.alert("You are using an old browser that does not allow to save data between session. Do not reload this page or you will lose your defined sequences.");
+            }
+
+
+            // Get data from local storage if it exists (we retain some data, like existing sequences and sequence numbers
+            // on page reload. Also works when the browser is closed and reopened.
+            if (localStorage.sequence_selector) {
```

Comment:
if `localStorage` is undefined then this will throw an exception, this syntax also will return true if the value is set to null so you will be assigning null to the variables again.